PackageInfo.g

```diff
@@ -70,14 +70,6 @@ ArchiveURL      := Concatenation( ~.SourceRepository.URL,
                                  "/", ~.PackageName, "-", ~.Version ),
 ArchiveFormats := ".tar.gz",
 
-AbstractHTML :=
-  "The <span class=\"pkgname\">kbmag</span> package is a\
-       <span class=\"pkgname\">GAP</span> interface to some `C' programs\
-   for running the Knuth-Bendix completion program on finite semigroup,\
-   monoid or group presentations, and for attempting to compute  automatic\
-   structures of finitely presented groups",
```

Comment:
`AbstractHTML` is a required field of every `PackageInfo.g`, so it must not be deleted.
  
It is used to generate https://www.gap-system.org/Packages/kbmag.html